### PR TITLE
Update T1218.yaml

### PR DESCRIPTION
**Pull request by Prakash22-k**
Test Name : System Binary Proxy Execution - Wlrmdr Lolbin Use as Launcher

description: Use wlrmdr(Windows Logon Reminder executable) as a proxy binary to evade defensive countermeasures

Tested the activity in Atomic Lab machines.

### CODE REVIEW

**Comment by NagaSivaGunturu** [comment]
@Prakash22-k, 
If there're no dependecies for this atomic test to run, you can remove the empty fields related to Dependency check:
  dependency_executor_name: 
  dependencies:

**Comment by Prakash22-k** [comment]
@NagaSivaGunturu , Thanks for checking. I've removed the empty fields and updated the commit.

File: atomics/T1218/T1218.yaml [hunk 1]
```diff
@@ -401,3 +401,19 @@ atomic_tests:
...
+  supported_platforms:
+  - windows
+  input_arguments:
+    payload_path:
+      description: Path to the executable
+      type: String
+      default: C:\Windows\System32\calc.exe
+  executor:
+    command: wlrmdr.exe -s 3600 -f 0 -t _ -m _ -a 11 -u "#{payload_path}"
+    cleanup_command:
```
Comment (cyberbuff) [inline atomics/T1218/T1218.yaml:416]: Can you remove the empty cleanup_command as well ?

**Review by cyberbuff — CHANGES_REQUESTED** [review]

File: atomics/T1218/T1218.yaml (hunk 1 above)
Comment (Prakash22-k) [reply atomics/T1218/T1218.yaml]: @cyberbuff , Apologies for the delay. As requested, I've removed the  empty cleanup_command from the atomics and updated the commit.

**Review by clr2of8 — APPROVED** [review]

**Pull request merged**